### DISCUSSION TITLE
WidgetBody font-size

### DISCUSSION THREAD

**Discussion by startshineye** [chatwoot/chatwoot#4631] [Contributors]
I want to Change  font size of message;
when I change  font-size from "--font-size-nano" to "--font-size-mini";It does not work.

```
.message-wrap {
      margin-right: var(--space-smaller);
      max-width: 100%;
      .chat-bubble {
        box-shadow: var(--shadow-medium);
        background: var(--color-woot);
        border-radius: var(--border-radius-large);
        color: var(--white);
        display: inline-block;
        font-size: var(--font-size-mini);
``` 

result:
<img width="356" alt="image" src="https://user-images.githubusercontent.com/19678129/167040335-aaa03eaa-e855-4865-a816-8e5e7b37f3f1.png">

**Comment by pranavrajs** [comment]
@startshineye Are you testing this in production or in development? 

If you are on production, you might need to rebuild the assets.

**Reply by startshineye** [reply]
@pranavrajs development
